README.md

```diff
@@ -252,6 +290,7 @@ vim.keymap.set("n", "<leader>glR", gitlab.revoke)
 vim.keymap.set("n", "<leader>glc", gitlab.create_comment)
 vim.keymap.set("v", "<leader>glc", gitlab.create_multiline_comment)
 vim.keymap.set("v", "<leader>glC", gitlab.create_comment_suggestion)
+vim.keymap.set("n", "<leader>glj", gitlab.jump_to_discussion_tree_from_diagnostic)
```

Comment:
I'd recommend `glm` (`move_to_discussion_tree_from_diagnostic`) as the default here to match the default `m` used to move to the diagnostic from the discussion tree. Easier to remember IMO.

Reply:
updated.